Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/water-hole/ansible-operator
```

Comment:
The command
  operator-sdk build quay.io/example/cdi-operator:v0.0.1
does almost this without the need of an embedded Dockerfile.
Do we prefer to maintain our own Docker file or to require the operator-sdk at build time?

Reply:
I don't think we need to.  The base Dockefile uses ansible runner, which enables tower integration and I don't think we need to customize the run command.